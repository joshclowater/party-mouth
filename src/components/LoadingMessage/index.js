import React, { Fragment } from 'react';
import Loading from 'src/components/Loading';

export default function(props) {
  return (
    <Fragment>
      <Loading />
      {props.message}
    </Fragment>
  );
}
