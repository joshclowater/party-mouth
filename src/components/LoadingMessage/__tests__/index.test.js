import React from 'react';
import { shallow } from 'enzyme';
import LoadingMessage from '../';

it('Renders correctly', () => {
  expect(shallow(<LoadingMessage message="mock message" />)).toMatchSnapshot();
});
