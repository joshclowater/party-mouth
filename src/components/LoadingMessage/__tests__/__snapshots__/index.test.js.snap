// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<Fragment>
  <styled.div />
  mock message
</Fragment>
`;
